IdentityCore/src/oauth2/MSIDExternalSSOContext.m

```diff
@@ -41,6 +49,12 @@ - (MSIDWPJKeyPairWithCert *)wpjKeyPairWithCertWithContext:(id<MSIDRequestContext
             return nil;
         }
         
+        if (self.loginManager.isDeviceRegistered && ![MSIDExternalSSOContext isPlatformSSORegisteredFlagSetInUserDefaults])
```

Comment:
Do we need to handle another else if !self.loginManager.isDeviceRegistered at all?

Reply:
+1

Reply:
fixed